src/util/mdUtil.ts

```
@@ -48,9 +54,39 @@ const turndownService = new TurndownService({
 turndownService.use(gfm);
 turndownService.use(tables);
 
+turndownService.addRule('strikethrough', {
+    filter: ['del', 's', 'strike'],
+    replacement: function (content:any) {
+        return '~~' + content + '~~'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)